tests/unit/test_id_cli.py

```diff
@@ -42,7 +44,7 @@ def mock_exists(monkeypatch: pytest.MonkeyPatch):
     monkeypatch.setattr('pathlib.Path.exists', lambda *a, **k: True)
 
 
-@pytest.fixture(scope='module')
+@pytest_asyncio.fixture(scope='module')
```

Comment:
Are you supposed to change these when the fixture doesn't do anything async?

Reply:
Should be harmless, but not necessary.

Reply:
It's just occurred to me that any `@pytest.fixture`s added on 8.4.x will likely make it into master unnoticed

Reply:
Yup.